### PR TITLE
Remove global parameters from auto-completions and fix autocompletion 

### PR DESCRIPTION
**Pull request by dhsathiya**
Fix on other repositories:
https://github.com/EasyEngine/auth-command/pull/44

Disable global parameters from being listed in auto completion.
```
--sites_path=
--locale=
--le-mail=
--wp-mail=
--color 
--no-color 
--debug=
--quiet 
```

Signed-off-by: dhsathiya <devarshisathiya5@gmail.com>

### CODE REVIEW

File: php/EE/Completions.php [hunk 1]
```diff
@@ -48,21 +52,20 @@ public function __construct( $line ) {
...
+			if ( 'ee' === $command->get_name() && false === $is_alias && false === $is_help ) {
 				$aliases = \EE::get_configurator()->get_aliases();
 				foreach ( $aliases as $name => $_ ) {
 					$this->add( "$name " );
 				}
 			}
-			foreach ( $command->get_subcommands() as $name => $_ ) {
-				$this->add( "$name " );
+			foreach ( $command->get_subcommands() as $name => $subcommand ) {
+				$this->add( $name . ':' . $subcommand->get_shortdesc() );
```
Comment (kirtangajjar) [inline php/EE/Completions.php:64]: @dhsathiya Is bash completion is also working properly after adding this?

Comment (dhsathiya) [reply php/EE/Completions.php:64]: no bash will not (not tested but will not). We have to add a shell check for that.
Pseudo:
```
   if zsh
       $this->add( $name . ':' . $subcommand->get_shortdesc() );
   else
       $this->add( $name );
```

Comment (dhsathiya) [reply php/EE/Completions.php:64]: Bash completion script also needs tweeking as we have site name in `arg #3`.
We can continue our deployment putting only Zsh completion for now.

**Comment by kirtangajjar** [comment]
Merged this PR in https://github.com/EasyEngine/easyengine/pull/1490

**Pull request closed without merging**